src/heliaServer.ts

```diff
@@ -41,7 +41,7 @@ export class HeliaServer {
   public routes: RouteEntry[]
 
   constructor (logger: debug.Debugger) {
-    this.log = logger.extend('fastify')
+    this.log = logger.extend('server')
```

Comment:
to differentiate between fastify logs vs `helia-http-gateway:server` and  `helia-http-gateway:server:helia-fetch`